### PR TITLE
ldns: also build example tools because they are very useful

### PR DESCRIPTION
**Pull request by Habbie**

### CODE REVIEW

File: Library/Formula/ldns.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Ldns < Formula
   homepage 'http://nlnetlabs.nl/projects/ldns/'
   url 'http://nlnetlabs.nl/downloads/ldns/ldns-1.6.17.tar.gz'
```
Comment (vszakats) [inline Library/Formula/ldns.rb]: Could you also update both `homepage` and `url` to use `https` protocol?

**Comment by Habbie** [comment]
Amended with https links

File: Library/Formula/ldns.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
 require 'formula'
 
 class Ldns < Formula
-  homepage 'http://nlnetlabs.nl/projects/ldns/'
-  url 'http://nlnetlabs.nl/downloads/ldns/ldns-1.6.17.tar.gz'
+  homepage 'https://nlnetlabs.nl/projects/ldns/'
+  url 'https://nlnetlabs.nl/downloads/ldns/ldns-1.6.17.tar.gz'
   sha1 '4218897b3c002aadfc7280b3f40cda829e05c9a4'
-  revision 1
+  revision 2
```
Comment (mikemcquaid) [inline Library/Formula/ldns.rb]: Leave the `revision` as-is, thanks.

**Comment by vszakats** [comment]
Thank you!

**Comment by Habbie** [comment]
Revision reverted. Shall I also address all other (older) complaints from `style` and `audit --strict`?

**Comment by mikemcquaid** [comment]
@Habbie If you're happy to: yeh, thanks!

**Comment by Habbie** [comment]
All old nits addressed; added a test based on one of the example tools. Not sure about the overly long lines, comments welcome as always.

**Comment by DomT4** [comment]
Are the examples installed automatically by `make install` here? They don't require any added formula dances to install?

**Comment by Habbie** [comment]
Yes, just the configure changes makes the building and installation of the examples work.

**Comment by DomT4** [comment]
Nice. That's relatively rare, usually we have to do a whole dance to fish the compiled examples out of buildpath :sweat_smile:.

**Comment by Habbie** [comment]
Yes, I was as surprised - last time I looked (years ago) there was no `--with-examples` in ldns indeed.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `ldns: build example tools.`.

**Pull request closed without merging**